README.md

```diff
@@ -1 +1,55 @@
+
+## Quest Creators
+
+In the context of BANANO Quest, to create a quest you need to send the the following to the Tavern smart contract:
+
+* Quest coordinates
+* Quest name
+* Quest hint
+* How many BANANOs you want the quest to give out
+* How much of an ETH prize to give out
+* The quadrant of space you want the quest to be in
+* The color of your quest
+
+Ethereum is a public, open source blockchain and it would be trivial for any person to query what the given coordinates answer to a quest are. We created a utility in the app where the client hashes all the possible coordinates into a merkle root and additional levels of the tree into the smart contract. This allows the contract to be able to verify whether the coordinates are a valid solution or not. While we are using coordinates for BANANO Quest, developers can use words, virtual world coordinates or any other set of answers that can be hashed into a merkle tree and put into a smart contract. 
+
+## Questers
+
+Questers are users of the application who wish to compete to complete  quests and earn BANANO tokens. Questers must have a nominal amount of Ether in their account to complete them. A quester chooses a quest they want to complete, read the hint associated with it, and msut travel to a physical area in the world to where they believe the quest can be completed. If correct when they submit the quest, an augmented reality banano shows up that they can take a screenshot and share with their friends.  
+
+# Tech Stack
+
+The Pocket team built a custom set of extensible tools to make it easier for developers who wish to build peer to peer applications. Pocket is blockchain agnostic, and everything is built with the goal of easily adding a plugin for any other blockchain into the ecosystem. By doing so, Pocket can be a simple, common interface for developers who want to use a specific cryptocurrency or multiple ones in their applications.  
+
+## Ethereum
+
+We chose Ethereum as the decentralized "backend" for BANANO Quest due to our familiarity and maturity of the tools and network. While confirmation times and lack of avoiding users having to use Ether is a significant UX problem, this is still an example application that hopes to inspire others into building more user friendly applications.   
+
+## Tavern smart contract
+
+Named after the classic location where gamers go and collect quests, Tavern is a smart contract that allows any Solidity developer to easily extend and make their own custom quest for their custom application. For more detailed information visit the repo for [Tavern](https://github.com/pokt-network/tavern).
+
+## Non-Fungible Tokens (NFT's)
```

Comment:
Can we add a section to explain the BananoToken ERC721? https://github.com/pokt-network/banano-token